src/components/selectable/selectable_templates/selectable_template_sitewide.tsx

```diff
@@ -153,6 +153,15 @@ export const EuiSelectableTemplateSitewide: FunctionComponent<EuiSelectableTempl
     setPopoverIsOpen(true);
   };
 
+  const onSearchKeydown = (e: React.KeyboardEvent<HTMLInputElement>) => {
+    searchProps && searchProps.onKeyDown && searchProps.onKeyDown(e);
+    if (canShowPopoverButton) return;
+
+    if (e.key === 'Enter') {
```

Comment:
It looks like we use a service for this instead of hard coding strings as a general repo-wide practice. On line 18 at the top of the file, import this from our `services` folder:

```
import { useCombinedRefs, throttle, keys } from '../../../services';
```

and on this line:
```suggestion
    if (e.key === keys.ENTER) {
```

Reply:
Fixed this commit.
https://github.com/elastic/eui/pull/5886/commits/896d716b26c513a031ebd4382fa861e14fb9f51e